src/robusta/integrations/rocketchat/sender.py

```diff
@@ -0,0 +1,409 @@
+import logging
+import os
+import tempfile
+from typing import Any, Dict, List, Optional
+
+from rocketchat_API.rocketchat import RocketChat
+
+from robusta.core.model.env_vars import SLACK_TABLE_COLUMNS_LIMIT
+from robusta.core.reporting.base import Emojis, Finding, FindingStatus
+from robusta.core.reporting.blocks import (
+    BaseBlock,
+    CallbackBlock,
+    CallbackChoice,
+    DividerBlock,
+    FileBlock,
+    HeaderBlock,
+    KubernetesDiffBlock,
+    LinkProp,
+    LinksBlock,
+    ListBlock,
+    MarkdownBlock,
+    ScanReportBlock,
+    TableBlock,
+)
+from robusta.core.reporting.callbacks import ExternalActionRequestBuilder
+from robusta.core.reporting.consts import EnrichmentAnnotation, SlackAnnotations
+from robusta.core.reporting.utils import add_pngs_for_all_svgs
+from robusta.core.sinks.rocketchat.rocketchat_sink_params import RocketchatSinkParams
+from robusta.core.sinks.transformer import Transformer
+
+ACTION_TRIGGER_PLAYBOOK = "trigger_playbook"
+ACTION_LINK = "link"
+RocketchatBlock = Dict[str, Any]
+MAX_BLOCK_CHARS = 3000
+
+
+class RocketchatSender:
+    def __init__(self, token: str, server_url: str, user_id: str, account_id: str, cluster_name: str, signing_key: str):
+        self.server_url = server_url
+        self.user_id = user_id
+        self.room_id = ""
+        """
+        Connect to Rocketchat and verify that the Rocketchat token is valid.
+        Return True on success, False on failure
+        """
+        self.rocketchat_client = RocketChat(user_id=user_id,
+                                            auth_token=token,
```

Comment:
Does the auth token has expiration?

Reply:
The auth tokens (Personal access tokens), I dont think they expire. [Stackoverflow](https://stackoverflow.com/a/45867251) answer